include/psp2kern/kernel/dmac.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * \usergroup{SceKernelDmacMgr}

```

Comment:
Also, the `Kernel` is [usually omitted](https://github.com/vitasdk/vita-headers/tree/master/include/psp2/kernel). Not sure this is a good idea to omit it, but I guess it's better to stick with conventions, and maybe open an issue to discuss the matter.